splunk-apm/README.md

```diff
@@ -0,0 +1,86 @@
+# Splunk distribution of OpenTelemetry Lambda
+
+Splunk OTEL lambda is a downstream distribution of opentelemetry-lambda. Supported OpenTelemetry Lambda layers are preconfigured for easy use with Splunk environment (ingest, SmartAgent). Users can enhance their existing Lambda functions by adding Splunk-managed layer directly. Layer ARNs [are published here](https://github.com/signalfx/lambda-layer-versions/).
+
+## Get started 
+
+Currently, following components are supported:
+- [Java wrapper](#java-wrapper) 
+
+### Configuration
+
+All wrapper layers come preconfigured for quick start with Splunk.
+
+1. Context propagation
+    By default, W3C specified Trace Context and Baggage (`tracecontext,baggage`) context propagation are used. 
+    
+    If you want to change this, set the `OTEL_PROPAGATORS` environment variable in your Lambda function code. For more information about available context propagators, see the [Propagator settings](https://github.com/open-telemetry/opentelemetry-java/tree/v1.1.0/sdk-extensions/autoconfigure#customizing-the-opentelemetry-sdk) for the OpenTelemetry Java.
+
+2. Traces export
+    By default, the OpenTelemetry Protocol (`otlp`) exporter is used.
+    If you want to use this exporter, set these environment variables in your Lambda function code:
+    ```
+    OTEL_EXPORTER_OTLP_ENDPOINT=<Endpoint URL>
+    SPLUNK_ACCESS_TOKEN="orgAccessToken"
+    ```
+    Splunk provides also token-authenticated `jaeger-thrift-splunk` exporter for customers that need to use that specific protocol. In order to use it, please set (example endpoint value for SmartAgent):
+    ```
+    OTEL_TRACES_EXPORTER=jaeger-thrift-splunk
+    OTEL_EXPORTER_JAEGER_ENDPOINT=<Endpoint URL>
+    SPLUNK_ACCESS_TOKEN="orgAccessToken"
+    ``` 
+    By default, `OTEL_EXPORTER_JAEGER_ENDPOINT` is set to `http://127.0.0.1:9080/v1/trace` (local SmartAgent endpoint)
+   
+    Default span flush wait timeout is 1 second. It is controlled with a following property (value in milliseconds):
+    ```
+    OTEL_INSTRUMENTATION_AWS_LAMBDA_FLUSH_TIMEOUT: 30000
+    ```
+3. OpenTelemetry Metrics export
+
+    By default, OTEL metrics are disabled.    
+    
+4. Sampling
+
+    By default `always_on` sampler is used, meaning that all spans are always sampled.
+
+5. Logging
+    
+    Logging is controller with `OTEL_LAMBDA_LOG_LEVEL` env property. Default set to `WARN`. When set to `DEBUG`, the logging exporter is added to traces exporter in order to help verify exported data.
```

Comment:
```suggestion
5. Logging
    
    Logging is controlled by the `OTEL_LAMBDA_LOG_LEVEL` environment variable. Default is `WARN`. When set to `DEBUG`, the logging exporter is added to traces exporter in order to help verify exported data.
```

Reply:
Could enabling `DEBUG` incur in increased costs in AWS? If that's the case, we should warn users.

Reply:
Good idea, will note this.